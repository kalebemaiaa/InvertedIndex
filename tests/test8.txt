QueryAnd(QueryOr(QueryWord("primeiro-ministro"), QueryWord("presidente")), QueryWord("governador"))
3115.raw
848.raw
1900.raw
12027.raw
4182.raw
3069.raw
4550.raw
5889.raw
3482.raw
11345.raw
3321.raw
2418.raw
2866.raw
231.raw
2219.raw
4813.raw
11446.raw
6209.raw
1187.raw
2283.raw
2890.raw
4988.raw
351.raw
1978.raw
4758.raw
90.raw
2807.raw
5883.raw
5486.raw
3918.raw
10941.raw
3209.raw
11321.raw
2170.raw
1709.raw
2235.raw
1153.raw
5575.raw
2835.raw
6262.raw
3653.raw
5967.raw
1006.raw
4720.raw
6295.raw
3861.raw
3262.raw
2901.raw
11070.raw
1226.raw
3682.raw
252.raw
4460.raw
5455.raw
3532.raw
4481.raw
3114.raw
5481.raw
11050.raw
1346.raw
1818.raw
3911.raw
11642.raw
8782.raw
1976.raw
3086.raw
638.raw
5020.raw
343.raw
11570.raw
10883.raw
4515.raw
4357.raw
5778.raw
1611.raw
5105.raw
11851.raw
2142.raw
3923.raw
1077.raw
3934.raw
2863.raw
11360.raw
1502.raw
6089.raw
821.raw
11511.raw
3259.raw
904.raw
11541.raw
4648.raw
6019.raw
11819.raw
3531.raw
882.raw
5157.raw
2002.raw
3609.raw
5073.raw
1865.raw
2918.raw
6039.raw
3413.raw
3338.raw
1645.raw
3223.raw
3926.raw
11748.raw
2683.raw
1249.raw
8736.raw
4286.raw
3604.raw
5377.raw
6369.raw
11688.raw
6256.raw
11185.raw
1888.raw
5576.raw
5709.raw
6185.raw
2914.raw
2359.raw
8831.raw
5689.raw
3600.raw
1819.raw
5934.raw
1017.raw
1937.raw
5226.raw
2052.raw
606.raw
4194.raw
414.raw
4738.raw
3467.raw
6265.raw
11779.raw
3148.raw
3832.raw
3427.raw
2398.raw
2303.raw
3931.raw
5502.raw
3868.raw
2635.raw
3213.raw
8820.raw
2224.raw
6297.raw
1234.raw
5014.raw
1608.raw
408.raw
2679.raw
1569.raw
6197.raw
5598.raw
2884.raw
10993.raw
2198.raw
5365.raw
3343.raw
5072.raw
825.raw
5868.raw
2666.raw
2700.raw
4766.raw
5727.raw
5936.raw
468.raw
11805.raw
3509.raw
10944.raw
11606.raw
3560.raw
4217.raw
230.raw
6298.raw
11415.raw
6330.raw
6115.raw
3100.raw
1141.raw
3738.raw
803.raw
3225.raw
10781.raw
520.raw
11744.raw
2878.raw
6306.raw
47.raw
11821.raw
11221.raw
3318.raw
