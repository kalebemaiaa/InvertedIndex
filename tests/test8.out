2
38
151
171
443
529
598
623
632
1275
1441
1503
1523
1568
1679
1808
1857
1936
1980
2092
2102
2923
3015
3040
3119
3202
3320
3325
3482
3499
3686
3711
3727
3877
3931
4058
4104
4195
4212
4214
4244
4261
4449
4479
4647
4741
4747
4844
4950
5031
5085
5398
5426
5504
5514
5599
6022
6222
6289
8751
8883
11406
11521
11606
